src/resolver.cpp

```diff
@@ -197,24 +197,13 @@ UsdOpenAssetIOResolver::UsdOpenAssetIOResolver() {
   // TODO(DF): Set appropriate locale.
```

Comment:
It's the `[Core] Remove debug logging` commit btw. Will approve without closing this, (and just let you close it once you're done.) because I assume it's something that's more convenient to get to in the final rebase, rather than doing another force push just for this.

Reply:
Fixed [here](https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/30/commits/0991cbf635864135a31d595859a773dde907b874).